docs/pages/advanced-usage/plugin.mdx

```diff
@@ -0,0 +1,99 @@
+# Advanced Usage: Plugin
```

Comment:
Can we make this just "HTCSS Plugin" or "HTCondor Plugin"? When this all gets rendered, the page is already displayed as being under the "Advanced Usage" section of docs, so I don't think we need that in the page title.